typescript/infra/config/environments/mainnet3/core/verification.json

```diff
@@ -1,2144 +1,2830 @@
 {
```

Comment:
the reason for this big diff is we moved to deterministic JSON stringifying in a previous PR, and this hadn't been applied yet to verification.jsons. I used https://jsonformatter.org/json-compare to make sure that the only value diff here is just the addition of `ancient8`